apps/src/sites/studio/pages/home/teacher_homepage.js

```diff
@@ -1,25 +1,42 @@
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import GradientNavCard from '@cdo/apps/templates/teacherHomepage/GradientNavCard';
+// import GradientNavCard from '@cdo/apps/templates/teacherHomepage/GradientNavCard';
+import CourseCard from '@cdo/apps/templates/teacherHomepage/CourseCard';
+import TeacherHomepage from '@cdo/apps/templates/teacherHomepage/TeacherHomepage';
 
 $(document).ready(showContent);
 
+function showContent() {
 
-const EXAMPLE_CARD_DATA = {
-  title: "Teacher Community",
-  description: "Ask questions about curriculum, share ideas from your lessons, and get help from other teachers",
-  image: "../../static/navcard-placeholder.png",
-  buttonText: "Connect Today",
-  link: "link to wherever"
-};
+  const teacherHomepageData = document.querySelector('script[data-teacherhomepage]');
+  const config = JSON.parse(teacherHomepageData.dataset.teacherhomepage);
 
-function showContent() {
+  const CourseCard1 = {
+    courseName: config.course1name,
+    description: config.course1description,
+    image: "this is where there will be the source for the photo",
+    link: config.course1link,
+    assignedSections: ["Section 1", "Section 2"]
+  };
+
+  const CourseCard2 = {
+    courseName: config.course2name,
+    description: config.course2description,
+    image: "this is where there will be the source for the photo",
+    link: config.course2link,
+    assignedSections: ["Section 1"]
+  };
 
   ReactDOM.render (
-    <GradientNavCard
-      cardData={EXAMPLE_CARD_DATA}
-    />,
+    <TeacherHomepage>
+      <CourseCard
+        cardData={CourseCard1}
```

Comment:
I think this has been asked (and possible answered) already in previous PRs, but is there a reason we're sending an opaque object (i.e. cardData) instead of individual props (courseName, description, image, etc.)